src/LoadEarlier.js

```diff
@@ -9,26 +11,19 @@ import {
   View,
   ViewPropTypes,
 } from 'react-native';
+import Colors from './Colors';
 
 export default class LoadEarlier extends React.Component {
   renderLoading() {
     if (this.props.isLoadingEarlier === false) {
-      return (
-        <Text style={[styles.text, this.props.textStyle]}>
-          {this.props.label}
-        </Text>
-      );
+      return <Text style={[styles.text, this.props.textStyle]}>{this.props.label}</Text>;
     }
     return (
       <View>
-        <Text style={[styles.text, this.props.textStyle, {
-            opacity: 0,
-          }]}>
-          {this.props.label}
-        </Text>
+        <Text style={[styles.text, this.props.textStyle, { opacity: 0 }]}>{this.props.label}</Text>
```

Comment:
These are pretty scrunched now, could we put `{this.props.label}` on a newline at least?

Reply:
Fixed